### PR TITLE
Added  to install.md how to run patient portal using Wamp

### PR DESCRIPTION
**Pull request by alpinus4**
I added instructions what is needed to change to run patient portal using Wamp.

patient portal solving

Fixes: https://github.com/LibreHealthIO/LibreEHR/issues/921

### CODE REVIEW

**Comment by alpinus4** [comment]
@teryhill

File: INSTALL.md [hunk 1]
```diff
@@ -238,6 +238,19 @@ Deny from all
 
 For proper access to all pages of the website, enable the `mod_rewrite` module by issuing `a2enmod rewrite` on a terminal.
 
+If you are using Wamp, to run patient portal is needed change the following code:
```
Comment (teryhill) [inline INSTALL.md]: Say this .          If you are running the patient portal these items have to be configured.
Add it to the FAQ's also.

Comment (alpinus4) [reply INSTALL.md]: Ok

Comment (teryhill) [reply INSTALL.md]: Under the FAQ say    I am getting a 500 error when trying to run portal dashboard. and then put the answer in that you use here

**Comment by alpinus4** [comment]
How about now? @teryhill

**Comment by aethelwulffe** [comment]
Three days with Two approvals and no downvotes.
Merging.

**Pull request merged**